simulator.py

```diff
@@ -46,7 +46,7 @@ def simulate(
     w,
     policy,
     deck_size=10000,
-    learn_span=365,
+    learn_span=10*365,
```

Comment:
It doesn't work because the learn_span is set here: https://github.com/open-spaced-repetition/SSP-MMC-FSRS/blob/0e09b79638e092569f5520935ac98fd99825cc35/script.py#L330